Packaging/Debian/nextspace-back-0.28.0+nextspace/debian/patches/debian-fonts.patch

```diff
@@ -0,0 +1,24 @@
+Subject: Search for fonts generated by mknfonts, Debian-specific
+
+* Source/art/FTFontEnumerator.m : Add /var/lib/GNUstep to the search path
+
+Forwarded: not-needed
+From: Hubert Chathi <uhoreg@debian.org>
+From: Yavor Doganov <yavor@gnu.org>
+Last-Update: 2016-07-04
+
+--- a/Source/art/FTFontEnumerator.m
++++ b/Source/art/FTFontEnumerator.m
+@@ -349,7 +349,11 @@
+   families_seen = [[NSMutableSet alloc] init];
+   families_pending = [[NSMutableSet alloc] init];
+ 
+-  paths = NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSAllDomainsMask, YES);
++  /* Search for fonts in all domains including those generated by
++     mknfonts in postinst.  */
++  paths = [NSSearchPathForDirectoriesInDomains (NSLibraryDirectory,
++						NSAllDomainsMask, YES)
++	      arrayByAddingObject: @"/var/lib/GNUstep"];
```

Comment:
In NEXTSPACE fonts are located in /Library/Fonts. Are you sure this change is correct?

Reply:
It's a left-over from debian's package, and it's likely wrong, thanks for pointing this out.

Reply:
so it's not really a problem in that /var/lib/GNUstep is added to the default list, but I'll remove it for a more consistent experience across distros.